data-app/gen_asr.py

```diff
@@ -55,7 +55,7 @@ def fetch_multiprocess(dept_dists:DepartmentDistribution,term:int) -> None:
 
 
 if __name__ == "__main__":
-    TERMS = [1225, 1229, 1233, 1235, 1239]
+    TERMS = [1229, 1233, 1235, 1239, 1243]
```

Comment:
Are we only keeping the last 5 terms? Data from the last 3 years seems appropriate, instead of the last 1.5 years. I could be reading this line wrong, let me know